x-pack/plugins/canvas/public/components/autocomplete/autocomplete.js

```diff
@@ -93,9 +94,9 @@ export class Autocomplete extends React.Component {
   selectNext() {
     const { items } = this.props;
     const { selectedIndex } = this.state;
-    if (selectedIndex >= 0 && selectedIndex < items.length - 1)
+    if (selectedIndex >= 0 && selectedIndex < items.length - 1) {
       this.setState({ selectedIndex: selectedIndex + 1 });
-    else this.setState({ selectedIndex: 0 });
+    } else this.setState({ selectedIndex: 0 });
```

Comment:
This violates the styleguide: https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#always-use-braces-for-conditionals-and-loops